docs/docs/configuration/secrets-manager.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 4
+---
+
+# How to use AWS Secret Manager, GCP Secret Manager, & Azure Key Vault
+
+Secrets from **AWS Secret Manager**, **GCP Secret Manager**, & **Azure Key Vault** can be used in any **Secure** field which looks like the following:
```

Comment:
Typo: both comma and ampersand : `**GCP Secret Manager**, & **Azure Key Vault**`